ocaml/xapi/repository.mli

```diff
@@ -12,7 +12,7 @@
  * GNU Lesser General Public License for more details.
  *)
 
-type guidance = RebootHost | RestartToolstack | EvacuateHost | RestartDeviceModel
+open Repository_helpers
```

Comment:
I think this adds everything from the helpers into the interface, which defeats part of the purpose of having an mli file. Instead, you could change below `Guidance.t` to `Repository_helpers.Guidance.t`.